backend/protos/campaign.proto

```diff
@@ -80,6 +83,22 @@ message CampaignTypesResponse {
 	repeated CampaignTypeDetails campaign_types = 1;
 }
 
+message CreateBeneficiaryRequest {
+
+	string name = 1;
+	common.nomenclatures.BeneficiaryType type = 2;
+	common.nomenclatures.ISO2CountryCode country_iso_code = 3;
+  string city = 4;
```

Comment:
formatting here can be fixed, seems that some of the fields are indented with tabs

Reply:
Done.